config/locales/searchworks.en.yml

```diff
@@ -25,10 +25,16 @@ en:
     search_dropdown:
       articles:
         label: articles+
-        description_html: <span class="h3">articles+</span> <span class="help-block">journal articles, e-books, and other licensed e-resources</span>
+        description_html: <span class="h3">articles+</span> <span class="help-block">journal articles & other e-resources</span>
```

Comment:
If this is html, should that be `&amp;` as in line 31?

Reply:
✅